services/report/parser/legacy.py

```diff
@@ -110,32 +111,27 @@ def parse_raw_report_from_bytes(self, raw_report: bytes) -> LegacyParsedRawRepor
             self.ignore_from_now_on_marker
         )
         sections = self.cut_sections(raw_report)
-        res = self._generate_parsed_report_from_sections(sections)
-        return res
+        return self._generate_parsed_report_from_sections(sections)
 
     def _generate_parsed_report_from_sections(self, sections):
         uploaded_files = []
-        toc_section = None
-        env_section = None
-        report_fixes_section = None
+        toc = None
+        report_fixes = None
         for sect in sections:
             if sect["footer"] == self.network_separator:
-                toc_section = sect["contents"]
+                toc = clean_toc(sect["contents"].decode(errors="replace").strip())
             elif sect["footer"] == self.env_separator:
-                env_section = sect["contents"]
+                pass
             else:
                 if sect["filename"] == "fixes":
-                    report_fixes_section = sect["contents"]
+                    report_fixes = sect["contents"]
                 else:
-                    uploaded_files.append(
-                        ParsedUploadedReportFile(
-                            filename=sect.get("filename"),
-                            file_contents=sect["contents"],
-                        )
+                    file = ParsedUploadedReportFile(
+                        filename=sect["filename"], file_contents=sect["contents"]
                     )
+                    uploaded_files.append(file)
         return LegacyParsedRawReport(
-            toc=toc_section,
-            env=env_section,
```

Comment:
Will we have to update documentation by removing this?
(Not against the removal, just trying to be helpful)

Reply:
which documentation do you mean?

Reply:
haha good point. I looked for docs mentioning this "env" options in the uploads but found none.

The only place you'd find is running `--help` on the `do-upload` command of the CLI honestly